### PR TITLE
Implement support for fetching hddtemp data

### PR DESCRIPTION
**Pull request by MendelGusmao**
Here's a basic implementation for getting temperature data from disks using hddtemp

### CODE REVIEW

File: plugins/inputs/hddtemp/hddtemp.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
+// +build linux,hddtemp
```
Comment (sparrc) [inline plugins/inputs/hddtemp/hddtemp.go]: why did you add an `hddtemp` build tag?

Comment (MendelGusmao) [inline plugins/inputs/hddtemp/hddtemp.go]: Got it. I'll update asap.

Comment (MendelGusmao) [inline plugins/inputs/hddtemp/hddtemp.go]: Fixed.

amend + push -f

**Comment by sparrc** [comment]
Looks good, but I noticed that go-hddtemp: https://github.com/MendelGusmao/go-hddtemp/blob/master/hddtemp.go is only ~60 lines of code.

Could you just copy hddtemp.go (and hddtemp_test.go) in-repo? You can add them with the license file in a directory, or put the license at the top of the file in a comment.

**Comment by MendelGusmao** [comment]
Fine, @sparrc ! Just pushed a modified commit according to your suggestions.

**Comment by sparrc** [comment]
Thanks @MendelGusmao. Am I correct in assuming that this requires that hddtemp is running as a daemon on the target system? Is this a common approach or would simply forking the `hddtemp` command and parsing the output suffice?

**Comment by MendelGusmao** [comment]
It's the common approach because hddtemp requires root access for gathering the data from the disks.

**Comment by sparrc** [comment]
this looks good, but you'll need to write a README: https://github.com/influxdata/telegraf/blob/master/CONTRIBUTING.md#steps-for-contributing

**Comment by MendelGusmao** [comment]
OK. I'll provide one.

**Comment by MendelGusmao** [comment]
Done!

**Comment by sparrc** [comment]
please rebase as well! 🙏

**Comment by MendelGusmao** [comment]
Done too.

**Comment by sparrc** [comment]
thanks @MendelGusmao
